docs/spark_processor.md

```diff
@@ -0,0 +1,22 @@
+# Spark Processor
+
+The SparkProcessor does feature ETL using Spark as the processing engine. In the
+following sections we describe the deployment modes supported by SparkProcessor
+and the configuration keys accepted by each mode.
+
+## Deployment Mode
+FeatHub's SparkProcessor only supports Spark's local mode. When running in local
```

Comment:
How does the user specify that the SparkProcessor should use local mode? If I understand correctly, the user must set the `master` to `local`. If that's the case how about making the default value of `master` `local`?